dvc/repo/diff.py

```diff
@@ -6,7 +6,7 @@
 
 
 @locked
-def diff(self, a_ref="HEAD", b_ref=None):
+def diff(self, a_rev="HEAD", b_rev=None):
     """
     By default, it compares the working tree with the last commit's tree.
```

Comment:
working tree -> current workspace in docstrings too? 🙂

Reply:
@jorgeorpinel Would rather not do that, as we use `WorkingTree` in the code a lot and that would break the symmetry.

Reply:
I totally understand. But we have asymmetry in terminology between core and docs e.g. this case (rev vs commit) working tree vs. workpsace, project vs repo... I'm not sure how big of a deal it is but we may need to address this at some point.

Reply:
@jorgeorpinel I agree, hence why we are changing docs and CLI/API. But changing code and comments is far from being as useful, so it can wait for now.